docs/6-infrastructure-configuration-management/6.1.2-terraform-backends.md

```diff
@@ -0,0 +1,67 @@
+# Terraform State & Backends
+
+## Sharing workspaces
+
+In the last tutorial from the previous section you learned how to use Terraform Cloud to store a Terraform state remotely. Using remote states makes it possible for teams to share and create automation for Terraform workspaces. Work with your teammates to in the following exercises to get a better understand of how to share a Terraform workspace.
```

Comment:
"Work with your teammates to in the following exercises to get a better understand of how..."

"Work with your teammates in the following exercises to get a better understanding of how..."
or
"In the following exercises, work with your teammates to get a better understanding of how..."